### PR TITLE
Try to fix autopkgtests

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: src/fu-util-common.c [hunk 1]
```diff
@@ -65,10 +65,14 @@ fu_util_get_expected_command(const gchar *target)
 gboolean
 fu_util_using_correct_daemon(GError **error)
 {
-#ifdef HAVE_SYSTEMD
 	g_autofree gchar *default_target = NULL;
 	g_autoptr(GError) error_local = NULL;
-	const gchar *target = fu_util_get_systemd_unit();
+	const gchar *target;
```
Comment (hughsie) [inline src/fu-util-common.c:70]: I think you've got to be a bit more careful here e.g. error_local, default_target are unused in the non HAVE_SYSTEMD case.

Comment (superm1) [reply src/fu-util-common.c]: fixed.  I forgot we just return TRUE in the non-systemd case

**Review by hughsie — DISMISSED** [review]
If it works, LGTM, thanks.

**Comment by superm1** [comment]
This seems to work for me now if that looks good.

**Pull request merged**